### PR TITLE
fix: print concrete error from fillPlugin in logs

### PR DESCRIPTION
**Pull request by randmonkey**
**What this PR does / why we need it**:

Print the concrete error returned from `fillPlugin` function in logs.

**Which issue this PR fixes**:

part of #5341 
**Special notes for your reviewer**:

REVIEW: how to write changelog for this?

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

**Comment by tao12345666333** [comment]
#5347 will fix the CI failure

File: internal/dataplane/deckgen/generate.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func ToDeckContent(
 			}
 			err = fillPlugin(ctx, &plugin, params.PluginSchemas)
 			if err != nil {
-				log.Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
+				log.Errorf("failed to fill-in defaults for plugin: %s, error %v", *plugin.Name, err)
```
Comment (czeslavo) [inline internal/dataplane/deckgen/generate.go:56]:
```suggestion
				log.WithError(err).Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
```

File: internal/dataplane/deckgen/generate.go [hunk 2]
```diff
@@ -71,7 +71,7 @@ func ToDeckContent(
 				}
 				err = fillPlugin(ctx, &plugin, params.PluginSchemas)
 				if err != nil {
-					log.Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
+					log.Errorf("failed to fill-in defaults for plugin: %s, error %v", *plugin.Name, err)
```
Comment (czeslavo) [inline internal/dataplane/deckgen/generate.go:74]:
```suggestion
					log.WithError(err).Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
```

File: internal/dataplane/deckgen/generate.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ func ToDeckContent(
 		}
 		err = fillPlugin(ctx, &plugin, params.PluginSchemas)
 		if err != nil {
-			log.Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
+			log.Errorf("failed to fill-in defaults for plugin: %s, error %v", *plugin.Name, err)
```
Comment (czeslavo) [inline internal/dataplane/deckgen/generate.go:98]:
```suggestion
			log.WithError(err).Errorf("failed to fill-in defaults for plugin: %s", *plugin.Name)
```

**Review by czeslavo — COMMENTED** [review]
Nits: we could use `WithError` to log the error as a log field.

**Review by czeslavo — APPROVED** [review]

**Pull request merged**